src/dust/dust.cpp

```diff
@@ -239,8 +372,13 @@ Real EstimateTimestepMesh(MeshData<Real> *md) {
   auto &dust_pkg = pm->packages.Get("dust");
   auto &params = dust_pkg->AllParams();
 
+  auto userDt = params.template Get<Real>("user_dt");
+  auto nspecies = params.template Get<int>("nspecies");
+
+  const auto cfl_number = params.template Get<Real>("cfl");
```

Comment:
There's no reason to modify this function anymore

Reply:
I can remove it

Reply:
fixed in the new code